tests/jest/components/tab/SearchInput.spec.js

```diff
@@ -0,0 +1,174 @@
+/* jshint esversion: 8 */
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import SearchInput from '../../../../src/components/tab/SearchInput'
+import workPackagesSearchResponse from '../../fixtures/workPackagesSearchResponse.json'
+
+jest.mock('@nextcloud/l10n', () => ({
+	translate: jest.fn((app, msg) => msg),
+	getLanguage: jest.fn(() => ''),
+}))
+
+const localVue = createLocalVue()
+
+describe('test search input', () => {
+	describe('state error', () => {
+		it('should be displayed when state is at error', async () => {
+			const wrapper = mount(SearchInput, {
+				localVue,
+				mocks: {
+					t: (msg) => msg,
+					generateUrl() {
+						return '/'
+					},
+				},
+			})
+			let errorSelector = wrapper.find('.title')
```

Comment:
```suggestion
			let errorSelector = wrapper.find(selectors.title)
```
use variables for the selectors. look for the other occurrences too.